release_notes/ocp-4-12-release-notes.adoc

```diff
@@ -722,81 +748,166 @@ In the table below, features are marked with the following statuses:
 |TP
 |TP
 
+|====
+
+[id="ocp-installation-technology-preview-features"]
+=== Installing technology preview features
+
+.Installing technology Preview tracker
+[cols="4,1,1,1",options="header"]
+|====
+|Feature |OCP 4.10 |OCP 4.11 | OCP 4.12
+
 |Adding kernel modules to nodes with kvc
 |TP
 |TP
 |TP
 
-|Non-preempting priority classes
+|Assisted Installer
 |TP
 |TP
 |TP
 
-|Assisted Installer
+|IBM Cloud VPC clusters
+|TP
+|TP
+|GA
+
+|Cloud controller manager for Alibaba Cloud
 |TP
 |TP
 |TP
 
-|`kdump` on `x86_64` architecture
+|Cloud controller manager for Amazon Web Services
+|TP
+|TP
 |TP
-|GA
-|GA
 
-|`kdump` on `arm64` architecture
-|-
+|Cloud controller manager for Google Cloud Platform
+|TP
 |TP
 |TP
 
-|`kdump` on `s390x` architecture
+|Cloud controller manager for Microsoft Azure
 |TP
 |TP
 |TP
 
-|`kdump` on `ppc64le` architecture
+|Cloud controller manager for {rh-openstack-first}
 |TP
 |TP
 |TP
 
-|IBM Cloud VPC clusters
+|Cloud controller manager for VMware vSphere
+|TP
 |TP
 |TP
-|GA
 
-|Serverless functions
+|Selectable Cluster Inventory
 |TP
 |TP
 |TP
 
-|Cloud controller manager for Alibaba Cloud
+|Heterogeneous clusters
```

Comment:
On the heterogeneous team, we are changing the wording from `heterogeneous clusters` to `multi-architecture cluster` in the docs. 

I'm working on making the change though out the books